### ISSUE TITLE
A Secretary?

### ISSUE THREAD

**Issue by gtewallace** [opencontainers/wg-freebsd-runtime#4]
Hi all. I'm noticing a few non-technical items that I'm happy to assist with if it would be useful, such as coordinating with the team on a call cadence, schedule, writing and publishing minutes, etc.

LMK

**Comment by samuelkarp** [comment]
I don't think we have a formal secretary role but assistance in that area would be appreciated!

**Comment by gtewallace** [comment]
sounds good. I will start by joining this week's OCI Developer's call and go from there.

**Comment by dfr** [comment]
Sorry for the late reply - I'm just getting over a nasty headcold (not covid, fortunately). I think the idea of a secretary is great and a regular meeting schedule is likely to help us make forward progress.

**Comment by gtewallace** [comment]
Sounds great. 

With the time zone spread (US Pacific to UTC) we're probably looking at a fairly narrow overlap at 8 or 9 Pacific, 11 or 12 Eastern, 4 or 5 UTC.

So, I suppose the question is, which day of the week does one of these times work for everyone? 

Here's a Doodle to indicate what works for you (I can do any day EXCEPT Tuesday at either of those times): https://doodle.com/meeting/participate/id/avmryOMe

Do folks think bi-weekly (every other week) is good to start?

Once we get this set up, I will update the ReadMe and also get us started with HackMD notes. 

Greg

**Comment by dch** [comment]
bi-weekly sounds good to me, my week evenings are already pretty cramped

**Comment by dfr** [comment]
Biweekly is a good place to start and we can adjust the frequency later if necessary. Most weeks, I can be available any weekday before 6pm GMT or a little later on Tue/Thu/Fri.

**Comment by mnour** [comment]
Hi @gtewallace. I would like to be invited to/attend the bi-weekly zoom call
